components/resources/library/src/commonMain/kotlin/org/jetbrains/compose/resources/StringResources.kt

```diff
@@ -0,0 +1,106 @@
+package org.jetbrains.compose.resources
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import org.jetbrains.compose.resources.vector.xmldom.Element
+import org.jetbrains.compose.resources.vector.xmldom.NodeList
+
+private const val STRINGS_XML = "strings.xml" //todo
+private val SimpleStringFormatRegex = Regex("""%(\d)\$[ds]""")
```

Comment:
We may create an issue in Kotlin core libraries to provide a common formatter

Reply:
https://youtrack.jetbrains.com/issue/KT-25506

Reply:
Left a comment there